### PR TITLE
kernel: Add new lts2019 kernel option

### PR DESCRIPTION
**Pull request by mdhorn**
Changes proposed in this pull request:
- Add newest lts kernel to choose list

### CODE REVIEW

**Comment by mdhorn** [comment]
Waiting for lts2019 bundle to be released and default lts switched.

**Comment by mdhorn** [comment]
The 2019 LTS bundle is available in the OS, so this PR is now unblocked.

_Note: The default LTS kernel is still the 2018 kernel, but that change will not be directly seen by the installer as it is only a bundle definition._

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 1]
```diff
@@ -662,6 +662,9 @@ msgstr "Kernel Linux de vanguardia adaptado a medida de metal"
 msgid "Run the latest Long Term Support (LTS) Linux Kernel"
 msgstr "Ejecute el último kernel de Linux de soporte a largo plazo (LTS)"
 
+msgid "Run the Long Term Support (LTS) 2019 Linux Kernel 5.4"
+msgstr "Ejecute el núcleo de soporte a largo plazo (LTS) 2019 Linux Kernel 5.4"
```
Comment (karthikprabhuvinod) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: Should we try to parameterize this?

msgid "Run the Long Term Support (LTS) %s Linux Kernel  %s"

File: locale/zh_CN/LC_MESSAGES/clr-installer.po [hunk 2]
```diff
@@ -662,6 +662,9 @@ msgstr "针对裸机定制的尖端 Linux 内核"
 msgid "Run the latest Long Term Support (LTS) Linux Kernel"
 msgstr "运行最新的长期支持 (LTS) Linux 内核"
 
+msgid "Run the Long Term Support (LTS) 2019 Linux Kernel 5.4"
+msgstr "运行长期支持 (LTS) 2019 Linux 内核 5.4"
```
Comment (karthikprabhuvinod) [inline locale/zh_CN/LC_MESSAGES/clr-installer.po]: same here

File: locale/es_MX/LC_MESSAGES/clr-installer.po (hunk 1 above)
Comment (mdhorn) [reply locale/es_MX/LC_MESSAGES/clr-installer.po]: I like the idea of parametrizing strings,  in this case, it doesn't work so well as we are loading the kernel descriptions from a JSON file and then the lookup is done based on that description:
https://github.com/clearlinux/clr-installer/blob/master/gui/pages/kernel_options.go#L93

File: locale/zh_CN/LC_MESSAGES/clr-installer.po (hunk 2 above)
Comment (mdhorn) [reply locale/zh_CN/LC_MESSAGES/clr-installer.po]: See above.

**Comment by mdhorn** [comment]
> If I am understanding it right, the kernel is packaged as an offline bundle which gets installed by swupd during the installation. Correct?

The kernel is just another bundle selection, but in this case, the user can selected something different from the default native kernel.

**Comment by mdhorn** [comment]
Fixed in clr-installer 2.4.2 first released in Clear Linux OS 32600

**Pull request merged**